### ISSUE TITLE
[require-doctype] should ignore when there is no html tag

### ISSUE THREAD

**Issue by yeonjuan** [yeonjuan/html-eslint#153]
For use in templates, this rule should not generate an error if `<html> <html>` is not present.

- AS-IS
```html
// error "Missing `<!DOCTYPE HTML>`"
<div>

</div>
```

- TO-BE

```html
// no-error
<div>

</div>
```
